.gitlab-ci.yml

```diff
@@ -17,7 +17,7 @@ include:
 
 .tmpl:deploy: &tmpl_deploy
   script:
-    - yarn sls deploy --verbose
+    - yarn sls deploy --stage=$SLS_STAGE --verbose
```

Comment:
Wasn't this being passed as environment variable, or you're just making it explicit?

Reply:
so this is really strange, i knew you would comment on it (and good thing you did) - serverless started throwing errors about missing opt.stage which i found really strange and this actually fixed it. I'll try to remove it and see if it makes a difference as I made many other changes since then so it might no longer be a problem

Reply:
guess it works now, go figure